eng/common/scripts/Delete-RemoteBranches.ps1

```diff
@@ -78,9 +119,12 @@ foreach ($res in $responses)
       exit 1
     }
   } 
+  
   try {
-    Remove-GitHubSourceReferences -RepoId $RepoId -Ref $branch -AuthToken $AuthToken
-    LogDebug "The branch [ $branchName ] in [ $RepoId ] has been deleted."
+    if (!$hasCentralPROpened -and $PSCmdlet.ShouldProcess("[ $branchName ] in [ $RepoId ]", "Deleting branches on cleanup script")) {
+      #Remove-GitHubSourceReferences -RepoId $RepoId -Ref $branch -AuthToken $AuthToken
+      LogDebug "The branch [ $branchName ] with sha [$($res.object.sha)] in [ $RepoId ] has been deleted."
```

Comment:
I believe ShouldProcess will log for you now so this is probably redundant.

Reply:
Totally agree. The reason why I miss Java branch in log is it did not print anything in my local. We could find issue earlier if we use Write-Host